docs/concepts/Auditing-Packages.md

```diff
@@ -0,0 +1,108 @@
+3. Type `dotnet restore`.
+4. Review the audit report and address the known security vulnerabilities.
+
+## Reviewing and acting on the security audit report
+
+Running `dotnet restore` will produce a report of security vulnerabilities with the affected package name, the severity of the vulnerability, and a link to the advisory for more details.
+
+### Security vulnerabilities found with updates
+
+If security vulnerabilities are found and updates are available for the package, you can either:
+
+- Edit the `.csproj` or other package version location(`Directory.Packages.props`) with a newer version containing a security fix.
+- Use the NuGet package manager user interface in Visual Studio to update the individual package.
+- Run the `dotnet add package` command with the respective package ID to update to the latest version.
+
+### Security vulnerabilities found with no updates
+
+In the case that a known vulnerability exists in a package without a security fix, you can do the following.
+
+- Check for any mitigating factors outlined in the advisory report.
+- Use a suggested package if the package is marked deprecated or is abandoned.
+- If the package is open source, consider contributing a fix.
+- Open an issue in the package's issue tracker.
+
+#### Check for mitigating factors
+
+Review the security advisor for any mitigating factors that may allow you to continue using the package with the vulnerability. The vulnerability may only exist when the code is used on a specific framework, operating system, or a special function is called.
+
+#### Use a suggested package
+
+In the case that a security advisory is reported for the package you're using and the package is marked deprecated or seems abandoned, consider using any suggested alternate package the package author has declared or a package comprising of similar functionality that is maintained.
+
+#### Contribute a fix
+
+If a fix does not exist for the security advisory, you may want to suggest changes that addresses the vulnerability in a pull request on package's open source repository or contact the author through the `Contact owners` section on the NuGet.org package detail page.
+
+#### Open an issue
+
+If you do not want to fix the vulnerability or are unable to update or replace the package, open an issue in the package's issue tracker or preferred contact method. On NuGet.org, you can navigate to the package details page and click `Report package` which will guide you to get in contact with the author.
+
+### No security vulnerabilities found
+
+If no security vulnerabilities are found, this means that packages with known vulnerabilities were not found in your package graph at the present moment of time you checked. Since the advisory database can be updated at any time, we recommend regularly checking your `dotnet restore` output and ensuring the same in your continuous integration process.
+
+### Setting a security audit level
```

Comment:
NuGetAuditMode is in the process of being added. Should we document this?  Or maybe not until we have a better design for transitive packages with known vulnerabilities?